crates/core_arch/src/aarch64/neon/generated.rs

```diff
@@ -0,0 +1,628 @@
+        assert_eq!(r, e);
+    }
+            
+    #[simd_test(enable = "neon")]
+    unsafe fn test_vcgeq_s64() {
+        let a:i64x2 = i64x2::new(1, 2);
+        let b:i64x2 = i64x2::new(0, 1);
+        let e:u64x2 = u64x2::new(0xFF_FF_FF_FF_FF_FF_FF_FF, 0xFF_FF_FF_FF_FF_FF_FF_FF);
+        let r: u64x2 = transmute(vcgeq_s64(transmute(a), transmute(b)));
+        assert_eq!(r, e);
+    }
+            
+    #[simd_test(enable = "neon")]
+    unsafe fn test_vcge_u64() {
+        let a:u64x1 = u64x1::new(1);
+        let b:u64x1 = u64x1::new(0);
+        let e:u64x1 = u64x1::new(0xFF_FF_FF_FF_FF_FF_FF_FF);
+        let r: u64x1 = transmute(vcge_u64(transmute(a), transmute(b)));
+        assert_eq!(r, e);
+    }
+            
+    #[simd_test(enable = "neon")]
+    unsafe fn test_vcgeq_u64() {
+        let a:u64x2 = u64x2::new(1, 2);
+        let b:u64x2 = u64x2::new(0, 1);
+        let e:u64x2 = u64x2::new(0xFF_FF_FF_FF_FF_FF_FF_FF, 0xFF_FF_FF_FF_FF_FF_FF_FF);
+        let r: u64x2 = transmute(vcgeq_u64(transmute(a), transmute(b)));
+        assert_eq!(r, e);
+    }
+            
+    #[simd_test(enable = "neon")]
+    unsafe fn test_vcge_f64() {
+        let a:f64 = 1.2;
+        let b:f64 = 0.1;
+        let e:u64x1 = u64x1::new(0xFF_FF_FF_FF_FF_FF_FF_FF);
+        let r: u64x1 = transmute(vcge_f64(transmute(a), transmute(b)));
+        assert_eq!(r, e);
+    }
+            
+    #[simd_test(enable = "neon")]
+    unsafe fn test_vcgeq_f64() {
+        let a:f64x2 = f64x2::new(1.2, 2.3);
+        let b:f64x2 = f64x2::new(0.1, 1.2);
+        let e:u64x2 = u64x2::new(0xFF_FF_FF_FF_FF_FF_FF_FF, 0xFF_FF_FF_FF_FF_FF_FF_FF);
+        let r: u64x2 = transmute(vcgeq_f64(transmute(a), transmute(b)));
+        assert_eq!(r, e);
+    }
+            
+    #[simd_test(enable = "neon")]
+    unsafe fn test_vmul_f64() {
+        let a:f64 = 1.0;
+        let b:f64 = 1.0;
+        let e:f64 = 1.0;
+        let r: f64 = transmute(vmul_f64(transmute(a), transmute(b)));
+        assert_eq!(r, e);
+    }
+            
+    #[simd_test(enable = "neon")]
+    unsafe fn test_vmulq_f64() {
+        let a:f64x2 = f64x2::new(1.0, 2.0);
+        let b:f64x2 = f64x2::new(1.0, 2.0);
+        let e:f64x2 = f64x2::new(1.0, 4.0);
+        let r: f64x2 = transmute(vmulq_f64(transmute(a), transmute(b)));
+        assert_eq!(r, e);
+    }
+            
+    #[simd_test(enable = "neon")]
+    unsafe fn test_vsub_f64() {
+        let a:f64 = 1.0;
+        let b:f64 = 1.0;
+        let e:f64 = 0.0;
+        let r: f64 = transmute(vsub_f64(transmute(a), transmute(b)));
+        assert_eq!(r, e);
+    }
+            
+    #[simd_test(enable = "neon")]
+    unsafe fn test_vsubq_f64() {
+        let a:f64x2 = f64x2::new(1.0, 4.0);
+        let b:f64x2 = f64x2::new(1.0, 2.0);
+        let e:f64x2 = f64x2::new(0.0, 2.0);
+        let r: f64x2 = transmute(vsubq_f64(transmute(a), transmute(b)));
+        assert_eq!(r, e);
+    }
+            }
```

Comment:
Something went off here. You probably want to pass the generated code through `rustfmt` before writing it to a file.